### DISCUSSION TITLE
Python and wand and Amazon Lambda

### DISCUSSION THREAD

**Discussion by 3rdxw** [ImageMagick/ImageMagick#7039] [Help]
### ImageMagick version

ImageMagick-7.0.8

### Operating system, version and so on

Amazon linux 2

### Description

I succeed In making the binaries and I want to use them in my Amazon Lambda, How can I achieve that? 

what should I write in my lambda_function.py
BTW some binaries got 0KB size

```
import os

# Set MAGICK_HOME and LD_LIBRARY_PATH environment variables
os.environ['MAGICK_HOME'] = '/result'
os.environ['LD_LIBRARY_PATH'] = '/result'


from wand.image import Image
from wand.drawing import Drawing
from wand.color import Color
```
but I end up with error:
Unable to import module 'lambda_function': MagickWand shared library not found.
You probably had not installed ImageMagick library

ImageMagick-7.0.8
Wand-0.6.13
python 3.8 and 3.10

I have already searched here and everywhere and followed every public guide but no luck!

**Comment by tylerapplebaum** [comment]
I'm struggling with the same issue.
